ClimsoftVer4/ClimsoftVer4/ucrFormDaily2.vb

```diff
@@ -363,7 +368,7 @@ Public Class ucrFormDaily2
                     If Not IsDBNull(row.Item("visUnits")) Then
                         rcdObservationInitial.visUnits = row.Item("visUnits")
                     End If
-                    'clsDataConnection.db.observationinitials.Add(rcdObservationInitial)
+                    clsDataConnection.db.observationinitials.Add(rcdObservationInitial)
```

Comment:
Have you tested this part?

Reply:
Actually that was a change that i had tried from another branch by mistake. The version from Ian has a bug i did not fix yet. 
Will share